cfgov/v1/models/base.py

```diff
@@ -217,6 +223,41 @@ def specific_categories_query(block, parent_slug):
         return {search_type: queryset for search_type, queryset in
                 related.items() if queryset}
 
+    def flush(self):
+        if not settings.ENABLE_AKAMAI_CACHE_PURGE:
+            logger.info(
+                'Page {slug} (id = {id}) was not invalidated because '
```

Comment:
Minor thing: because this file doesn't have `from __future__ import unicode_literals` you should use unicode strings (`u'Page {slug}...'`) in case slugs contain unicode characters. Similarly at the other logging statement [below](https://github.com/cfpb/cfgov-refresh/pull/2801/files#diff-8e8a3e28d46ee12516bea4f5b50bf5a4R252).